pom.xml

```diff
@@ -7,6 +7,45 @@
 
   <build>
     <plugins>
+
+      <plugin>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <environmentVariables>
+            <CACHE_POOL_SIZE>10</CACHE_POOL_SIZE>
+            <CACHE_SERVER>localhost:1234</CACHE_SERVER>
+            <COOKIE_DOMAIN>.companieshouse.gov.uk</COOKIE_DOMAIN>
+            <COOKIE_NAME>abc</COOKIE_NAME>
+            <COOKIE_SECRET>iamasecret</COOKIE_SECRET>
+            <DEFAULT_SESSION_EXPIRATION>3600</DEFAULT_SESSION_EXPIRATION>
+            <HUMAN_LOG>1</HUMAN_LOG>
```

Comment:
Are the values correct? Doesn't seem like something we would put into live?

Reply:
The Surefire plugin only applies to unit tests. They are will not be live values.

Reply:
The values were take directly from the session handler as well.

Reply:
👍